core/rawdb/accessors_chain_test.go

```diff
@@ -276,17 +378,12 @@ func TestBlockReceiptStorage(t *testing.T) {
 	receipts := []*types.Receipt{receipt1, receipt2}
 
 	// Check that no receipt entries are in a pristine database
-	header := &types.Header{Number: big.NewInt(0), Extra: []byte("test header")}
-	hash := header.Hash()
+	hash := common.BytesToHash([]byte{0x03, 0x14})
```

Comment:
This is original geth code, testing less things - is this ok to remove the additional checks we had added? 🤔 I suppose worth it anyway, but I wanted to double check still.

Reply:
seems fine, if in doubt we can replicate the tests in an `ext` package.